install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
+
+# Check if docs-builder already exists
+if [ -f /usr/local/bin/docs-builder ]; then
+  echo "docs-builder is already installed."
+  printf "Do you want to update/overwrite it? (y/n): "
+  read choice
+  case "$choice" in
+    y|Y ) echo "Updating docs-builder..." ;;
+    n|N ) echo "Installation aborted."; exit 0 ;;
+    * ) echo "Invalid choice. Installation aborted."; exit 1 ;;
+  esac
+fi
+
+# Download the latest macOS binary from releases
+curl -LO https://github.com/elastic/docs-builder/releases/latest/download/docs-builder-mac-arm64.zip
```

Comment:
Should we add something more dynamic?

like identifying the OS and arch?

Reply:
That would be ideal, especially for Linux. Let me give that a try based on cat /etc/os-release